tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_ms.py

```diff
@@ -82,9 +68,16 @@ def check_automated_recovery_from_stopped_node(nodes):
     4) The new osd pods with the same ids should start on the stopped node after it powered on.
 
     """
+    old_wnodes = get_worker_nodes()
+    log.info(f"start worker nodes: {old_wnodes}")
```

Comment:
I think this list give the name of worker nodes currently running. The log "start worker nodes" looks like an action to start these nodes.

Reply:
Okay, I will change the log message.